lib/Support/ModuleInterpreter.cpp

```diff
@@ -361,6 +365,13 @@ void ModuleInterpreter::invoke(bool express_type) {
   case mem_mode_t::PART_TENSOR_IN_MEM:
     invoke_part_in_mem(express_type);
     break;
+  case mem_mode_t::ALL_TENSOR_IN_DISK:
```

Comment:
line 37-47 can not set mem_mode to ALL_TENSOR_IN_DISK, so this branch cannot be executed?

Reply:
Yes, there is no corresponding case  here. I modified the mem_ mode=mem_ mode_t: : ALL_ TENSOR_ IN_ DISK for testing, perhaps a threshold greater than 16GB is needed to activate the case. I don't know what the threshold is, but I hope you can set it. In the above test, it was passed.